ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1222,6 +1231,7 @@ func (r *RayClusterReconciler) calculateStatus(ctx context.Context, instance *ra
 		return nil, err
 	}
 
+	newInstance.Status.ReadyWorkerReplicas = utils.CalculateReadyReplicas(runtimePods)
```

Comment:
Just for my own understanding, we can't use the existing `AvailableWorkerReplicas` field because that only counts "running" pods and not necessarily "ready" pods, is that correct?

Reply:
yes